section2/exercises/if_statements.rb

```diff
@@ -63,3 +66,9 @@
 
 cups_of_flour = 1
 has_sauce = true
+
+if cups_of_flour >= 2 && has_sauce == true
```

Comment:
Improvement item - This line is the same thing as `if cups_of_flour >= 2 && has_sauce` - and this is more concise!